### PR TITLE
Add Next.js.

### PR DESCRIPTION
**Pull request by leerob**
Thank you 🙏

### CODE REVIEW

**Comment by jamiebuilds** [comment]
This is a verified server so you already have some features turned on for your server. But I've gone ahead and enabled the [Server Banner](https://support.discord.com/hc/en-us/articles/360028716472-Server-Banner-Background-Invite-Splash-Image) feature for you.

File: communities.json [hunk 1]
```diff
@@ -467,6 +467,13 @@
       "quote": "Discord allows Nuxt.js core team to chat and take decisions together in real-time. It engages our open source community to ask questions, get feedbacks for their projects and learn from each others.",
       "githubUrl": "https://github.com/nuxt/nuxt.js"
     },
+    {
+      "logo": "nextjs.svg",
+      "title": "Next.js",
+      "inviteCode": "yHzQeKFUcw",
```
Comment (jamiebuilds) [inline communities.json:473]: It looks like your server has a vanity invite code `nextjs` would you rather set it to that? Not sure if this invite code has any particular settings you want to keep, but non-vanity invite codes often have an expiration date

Comment (leerob) [reply communities.json:473]: Yeah, might as well use that I'd say 👍

Comment (leerob) [inline communities.json:473]:
```suggestion
      "inviteCode": "nextjs",
```

**Comment by leerob** [comment]
> This is a verified server so you already have some features turned on for your server. But I've gone ahead and enabled the Server Banner feature for you.

Thank you so much!

**Review by jamiebuilds — APPROVED** [review]

**Comment by leerob** [comment]
Is there anything else needed from me to get this through? 🙏

**Comment by leerob** [comment]
Closing this for now, but happy to open it back up if y'all are interested in accepting 😄

**Pull request closed without merging**